### PR TITLE
Add Exposed to interfaces.

### PR DESCRIPTION
**Pull request by rmcilroy**
Addresses #64

### CODE REVIEW

**Comment by rmcilroy** [comment]
@igrigorik PTAL, thanks.

**Review by igrigorik — COMMENTED** [review]
LGTM, modulo.. missing spaces between [] and dfn?

@annevk quick sanity check, before we merge? :)

**Comment by rmcilroy** [comment]
Fixed the whitespace

File: index.html [hunk 1]
```diff
@@ -324,15 +324,15 @@
     [[!HTML5]]</p>
     <pre class='idl'>
 partial interface Window {
-  unsigned long requestIdleCallback(IdleRequestCallback callback, optional IdleRequestOptions options);
-  void cancelIdleCallback(unsigned long handle);
+  [Exposed=Window] unsigned long requestIdleCallback(IdleRequestCallback callback, optional IdleRequestOptions options);
+  [Exposed=Window] void cancelIdleCallback(unsigned long handle);
```
Comment (annevk) [inline index.html]: These two are unneeded as `Window` itself is already restricted.

File: index.html [hunk 2]
```diff
@@ -324,15 +324,15 @@
...
 };
 
-dictionary IdleRequestOptions {
+[Exposed=Window] dictionary IdleRequestOptions {
```
Comment (annevk) [inline index.html]: This is not allowed (and unneeded).

**Review by annevk — CHANGES_REQUESTED** [review]
You went a little overboard. 😊 It's just interfaces that need this.

File: index.html (hunk 1 above)
Comment (rmcilroy) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (rmcilroy) [reply index.html]: Done

**Comment by rmcilroy** [comment]
Thanks, removed the unnecessary exposed attributes.

**Review by annevk — APPROVED** [review]

**Comment by rmcilroy** [comment]
Thanks. Committing.

**Comment by annevk** [comment]
I don't see it in the commit log...

**Comment by rmcilroy** [comment]
Sorry pressed the wrong button, done now.

**Comment by igrigorik** [comment]
Thanks Ross!

**Pull request merged**